src/app/Fake.DotNet.FxCop/FxCop.fs

```diff
@@ -0,0 +1,212 @@
+[<RequireQualifiedAccess>]
+module Fake.DotNet.FxCop
```

Comment:
(optional) I often get the feedback that actual samples help a lot so adding one to the docs here would be nice. Just a simple example like the following is enough:

https://github.com/fsharp/FAKE/blob/3827f95ad533978ec14c29db92658f48b0609e05/src/app/Fake.Api.GitHub/GitHub.fs#L11-L32